listener/upload_listener.py

```diff
@@ -327,7 +328,12 @@ def process_upload(upload_data, loop=None):
                                          vmaas_request, repo_list)
         # only give evaluator work if the system's vmaas-call has changed since the last time we did this
         if ImportStatus.CHANGED in import_status or DISABLE_OPTIMISATION:
-            new_upload_msg = {"type": "upload_new_file", "inventory_id": upload_data["id"]}
+            new_upload_msg = {
+                "type": "upload_new_file",
+                "inventory_id": upload_data["id"],
+                "request_id": upload_data["request_id"],
```

Comment:
shouldn't request_id be payload_id?

Reply:
No, `send_msg_to_payload_tracker` handles it 
```suggestion
                "request_id": upload_data["request_id"],
```